sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -1109,6 +1111,8 @@ class HistGradientBoostingClassifier(BaseHistGradientBoosting,
     ----------
     classes_ : array, shape = (n_classes,)
         Class labels.
+    do_early_stopping_ : bool
```

Comment:
We'll also need this for the regressor

Reply:
Right, @simonamaggio is taking care of that in #17677.

Reply:
it'd be easier to have everything in the same PR so that we can ensure consistency, but OK